test/test_history.py

```diff
@@ -19,23 +18,29 @@ class HistoryTest(unittest.TestCase):
     lb: np.ndarray = None
 
     def check_history(self):
-        self.obj.history.options = self.obj.options
-
         self.problem = pypesto.Problem(self.obj, self.lb, self.ub)
 
         optimize_options = pypesto.OptimizeOptions(
             allow_failed_starts=False
         )
 
+        history_options = pypesto.OptimizerHistoryOptions(
+            trace_record=True,
+            trace_record_hess=False,
+            trace_save_iter=1,
+            storage_file='/tmp/traces/conversion_example_{id}.csv',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Probable insecure usage of temp file/directory.](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5265302)